src/homeWork10/FileLoggerConfigurationLoader.java

```diff
@@ -0,0 +1,31 @@
+package homeWork10;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class FileLoggerConfigurationLoader {
+
+    public static FileLoggerConfiguration load() {
+
+        String fileName;
+        String fileFormat;
+        LoggingLevel loggingLevels;
+        int fileMaxSize;
+
+        try (InputStream is = new FileInputStream("config.properties")) {
+            Properties prop = new Properties();
+            prop.load(is);
+
+            fileName = prop.getProperty("FILE");
+            fileFormat = prop.getProperty("FORMAT");
+            loggingLevels = LoggingLevel.valueOf(prop.getProperty("LEVEL").toUpperCase());
```

Comment:
Why did you use `.toUpperCase()` here? It would be better to pass `DEBUG` in your `config.properties`, and stop doing this operation